src/api.py

```diff
@@ -154,7 +155,7 @@ def full_async(library_key, tag=None, brief=False, assume_latest=False):
       versions = yield versions_future
       result['versions'] = versions
       if len(versions) > 0:
-        result['latest_version'] = versions[-1]
+        result['latest_version'] = versiontag.default_version(versions)
```

Comment:
Should we add `default_version` to the metadata too and add a comment to remove `latest_version`?